client/src/index.css

```diff
@@ -38,6 +38,13 @@ s {
 
     color: white;
 }
+.timer-box{
+    width: 100%;
+    height: .5rem;
+    margin: 0%;
+    padding: 0%;
+    background-color: #220000;
+}
```

Comment:
This is never used -- though it maybe should be

Reply:
you are wrong